### PR TITLE
[1LP][RFR] Update miq sprout options

### PR DESCRIPTION
**Pull request by mshriver**
Add required options for SproutProvisioningRequest

Log commands that are run by dockerbot setup script

### CODE REVIEW

File: cfme/scripting/sprout.py [hunk 1]
```diff
@@ -41,14 +41,20 @@ def main():
 @click.option('--populate-yaml', is_flag=True, default=False,
               help="Populate the yaml with the appliance")
 @click.option('--provider', default=None, help="Which provider to use")
+@click.option('--provider_type', default=None, help='A provider type to select from')
```
Comment (jamesooden) [inline cfme/scripting/sprout.py]: here you use an underbar as a delimiter and for --template-type you use a -.    Just need to be consistent.   BTW, if you want --provider-type to use the variable provider_type, then your line would look like:

    `@click.option('--provider-type', 'provider_type', default=None, help='A provider type to select from')`

File: cfme/scripting/sprout.py [hunk 2]
```diff
@@ -41,14 +41,20 @@ def main():
...
+@click.option('--template-type', default=None, help='A template type')
```
Comment (jamesooden) [inline cfme/scripting/sprout.py]: This should look like:

`@click.option('--template-type', 'template_type', default=None, help='A template type')`

Based on line 48.

File: cfme/scripting/sprout.py (hunk 1 above)
Comment (jamesooden) [reply cfme/scripting/sprout.py]: This is described here:  http://click.pocoo.org/5/parameters/#parameter-names

File: cfme/scripting/sprout.py [hunk 3]
```diff
@@ -41,14 +41,21 @@ def main():
 @click.option('--populate-yaml', is_flag=True, default=False,
               help="Populate the yaml with the appliance")
 @click.option('--provider', default=None, help="Which provider to use")
+@click.option('--provider-type', 'provider_type', default=None,
+              help='A provider type to select from')
+@click.option('--template-type', 'template_type', default=None, help='A template type')
+@click.option('--preconfigured', default=True, help='Whether the appliance is configured')
```
Comment (izapolsk) [inline cfme/scripting/sprout.py]: Question: will this option have False if it is used ?

Comment (mshriver) [reply cfme/scripting/sprout.py]: @izapolsk Good call, I'll actually add a --notconfigured false boolean, so its an explicit flag.

**Pull request merged**